regl-camera.js

```diff
@@ -41,10 +42,14 @@ function createCamera (regl, props_) {
   var prevY = 0
 
   if (isBrowser && props.mouse !== false) {
-    mouseChange(function (buttons, x, y) {
+    var source = element || window
+    var width = element ? element.offsetWidth : window.innerWidth
+    var height = element ? element.offsetHeight : window.innerHeight
+    var noScroll = true
```

Comment:
Thanks, yeah I'm not really sure under what circumstances you would want this to be `false` (I guess if you had a full viewport element it might enable scroll-blocking perhaps?)